src/main/java/controllers/kvis/KvisAPIController.java

```diff
@@ -21,139 +20,121 @@
 import static common.AccessScope.playerScope;
 
 @Path("kvis")
-public class KvisAPIController
```

Comment:
Hvad er der sket her? xD 

Jeg kan ikke se hvad der er lavet, da den af en eller anden grund viser at det hele er blevet slettet og skrevet igen :smile:

Reply:
Det kan jeg ikke forklare. Ændringerne er i metoden for activateKvis, hvor return string står i "", så den kan læses som et json objekt, og i øvrigt returnerer "in use" ved 409-fejl. Det er kun den metode, der er ændret.

Reply:
Jeg merger ind fra dev i mit, så der er en clean ApiController